filesystem/filesystem.go

```diff
@@ -147,6 +149,11 @@ func (c *FileSystem) UnMountRootfs(cluster *v1.Cluster) error {
 
 func mountRootfs(ipList []string, target string, cluster *v1.Cluster) error {
 	SSH := ssh.NewSSHByCluster(cluster)
+	d := runtime.Default{
```

Comment:
Using the runtime struct seems not very good, other package only care about the runtime interface.

I think we can define the function GetRegistryConfig(defaultIP string), so you don't need the runtime.Default struct here.

Reply:
OK